docs_user/assemblies/assembly_migrating-databases-to-the-control-plane.adoc

```diff
@@ -6,6 +6,15 @@ ifdef::context[:parent-context: {context}]
 
 = Migrating databases to the control plane
 
+ifeval::["{OpenStackPreviousInstaller}" != "director_operator"]
+:pod_ns: '"${RHOSO_NAMESPACE}"'
+endif::[]
+ifeval::["{OpenStackPreviousInstaller}" == "director_operator"]
+:pod_ns: '"${OSPDO_NAMESPACE}"'
+endif::[]
+
```

Comment:
@pinikomarov @jistr I'm still not sure I understand why this text was put here. Is it because you needed to declare these values somewhere? Is there a better place for this, such as a separate attributes file or in Makefile?

Reply:
ok , a long the ospdo version gets the correct namespace logic I'm good